### PR TITLE
#211 meet button landing page

### PR DESCRIPTION
**Pull request by r-mishev**
Added id to members-section-container and made the button redirect to it. Changed the header of the container.

### CODE REVIEW

**Comment by asynchroza** [comment]
What about this [discussion](https://github.com/AUBGTheHUB/spa-website-2022/issues/211#issuecomment-1419172579)? @SuelaCark @r-mishev

**Review by asynchroza — APPROVED** [review]
LGTM! Just decide upon what should the text in the header be.

**Comment by SuelaCark** [comment]
@r-mishev I agree that we should keep the title "Hubbers" only for the header of the section. Do not change the button "Meet the team," leave it as it is.

File: packages/web/src/components/spa/MembersSection/MembersSection.jsx [hunk 1]
```diff
@@ -46,8 +46,9 @@ export const MembersSection = () => {
                 style={{
                     transform: `scale(${isMobile && !isFoldRes ? 1.1 : 1})`
                 }}
+                id="team"
             >
-                <h1 className="header-for-container">Hubbers</h1>
```
Comment (SuelaCark) [inline packages/web/src/components/spa/MembersSection/MembersSection.jsx:50]: Leave here the title section as "Hubbers"

**Pull request merged**